pages/common/git-delete-branch.md

```diff
@@ -0,0 +1,13 @@
+# git delete-branch
+
+> Delete local and remote Git branches.
+> Part of `git-extras`. When deleting a checked out branch, only the remote branch will be deleted.
```

Comment:
There can only be one checked out branch, so I'd adjust the sentence like so:
```suggestion
> Part of `git-extras`. If deleting the checked out branch, only the remote branch will be deleted.
```

Reply:
I reallly dislike the usage of the `the`, it makes more nouny rather than the verb  status it deserves. Its start to explain :smile:

Reply:
I'm not really sure what you mean; `a` and `the` are both articles and the only difference between them is the number of the nouns that they're used with.  Using "the" is nothing but more correct here.

Reply:
https://github.com/tldr-pages/tldr/pull/6011#discussion_r637851077 @marchersimon seemed to agree with me. Need a more solid opinion rather than +1

Reply:
I also think `the` makes more sense here, since there's only *the* one checked out branch. However `a` is fine for me too and it's still your PR.

Reply:
Committedd